parachain/runtime/src/lib.rs

```diff
@@ -359,11 +359,18 @@ impl dispatch::Config for Runtime {
 	type CallFilter = CallFilter;
 }
 
+parameter_types! {
+	pub RewardsAccount: AccountId = DotModuleId::get().into_account();
```

Comment:
Do we really want to use the same account for rewards as for locking of DOT?

Reply:
I thought about this too. I agree its a bit weird. My thinking was that relayers are paid instantaneously on receipt of each incoming message. So the fee will be deducted from the user's account and find its way into the relayers account within a single transaction. That means having an intermediary rewards account to hold collected fees would be kinda redundant.

But I think I will adding an intermediary rewards account is still a good thing especially as we may want to allocate fees differently (ie. relayers only get 80% of the fee charged to a user).

Will make the change.

Reply:
Actually I'm going to leave this one for the next related piece of work that needs to happen - Governance. We'll be reworking rewards/fee allocation in that task anyway.